README.md

```diff
@@ -17,16 +17,16 @@ Get your dev environment set up (PC or Mac):
 * [Install Node](https://nodejs.org/en/download/)
 * Choose your favorite IDE, e.g. [Visual Studio Code](https://code.visualstudio.com/).
 
-Clone the CLI repo
+Install the CLI module
 
 ```bash
-$ git clone https://github.com/openT2T/opent2t-cli.git
```

Comment:
Can the CLI still be used without installing it as it is today?  If I npm install my translators in the opent2t-cli it should still work right?